### PR TITLE
Add note about BSD flags to changelog. Fixes #4246

### LINKED ISSUE #4246: much slower backups after upgrade from 1.0.9 to 1.1.8
## Have you checked borgbackup docs, FAQ, and open Github issues?
Yes

## Is this a BUG / ISSUE report or a QUESTION?
Issue

## System information. For client/server mode post info for both machines.
borg 1.0.9 (Debian Jessie) [Source] / borg 1.0.9 (Debian Stretch) [Repo via SSH)
upgrade to
borg 1.1.8 (Binary) [Source] / borg 1.1.8 (Debian Stretch Backports) [Repo via SSH)

#### Hardware / network configuration, and filesystems used.
ext3 [Source]
SSH
ext4 [Repo]

#### How much data is handled by borg?
Files: 554747 / 175.21 GB

#### Full borg commandline that lead to the problem (leave away excludes and passwords)
borg create -C lz4 --exclude-caches --exclude-from "${BACKUP_EXCLUDE}" -v -s borg@${SERVER}:data::'{hostname}{utcnow:%Y-%m-%d%H%M%S}' /source (3 dirs)

## Describe the problem you're observing.
First run after switch from 1.0.9 to 1.1.8.
Duration: 4 hours 18 minutes 11.14 seconds / Files: 554747 / 175.21 GB
Chunks: 1039359 unique/56809737 total

1.0.9 = Duration: 11 minutes 15.39 seconds / 554742 files
1.1.8 = Duration: 2 hours 29 minutes 17.52 seconds / 554358 files

I had two source jobs, /data (the log above) and /data/group (a subset of the data) and push to the same repo. Subset borg job goes from 2 mins to 25 mins.

maybe related:
https://github.com/borgbackup/borg/issues/2245
https://github.com/borgbackup/borg/issues/3239

### PR DESCRIPTION
**Pull request by m3nu**

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -545,6 +545,9 @@ Compatibility notes:
   unexpected) was rather confusing and unexplainable at first to users.
   If you want that "do not backup NODUMP-flagged files" behaviour, use:
   borg create --exclude-nodump ...
+- If you are on Linux and have many small files, consider using ``--nobsdflags``
```
Comment (ThomasWaldmann) [inline docs/changes.rst]: ... and you do not need bsdflags archived, consider ...

File: docs/changes.rst [hunk 2]
```diff
@@ -545,6 +545,9 @@ Compatibility notes:
...
+  with ``borg create`` to avoid additional syscalls and speed up backup creation.
+  Not all Linux filesystems fully support BSD flags so you may not need them.
```
Comment (ThomasWaldmann) [inline docs/changes.rst]: I'ld kill that last sentence.

It does not matter whether bsdflags are fully supported by the fs, but rather whether the user uses the ones which are supported and wants to have them in the backup archive.

**Comment by ThomasWaldmann** [comment]
you added it to the 1.1.1 changelog entry, was that the time when we added bsdflags? or was it 1.1.0?

**Comment by m3nu** [comment]
I added it to [1.1.1](https://borgbackup.readthedocs.io/en/stable/changes.html#version-1-1-1-2017-10-22) because there the option `--nobsdflags` was added. I guess processing of BSD flags was added before that, but no choice to disable them.

**Comment by m3nu** [comment]
Updated wording here too.

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**